docs/en/advanced-concepts/settings.md

```diff
@@ -38,3 +38,5 @@ Settings are implemented via the following API endpoints:
 ### Supported Settings
 
 **"ignoreUnimportantViews"** - Boolean which sets whether Android devices should use `setCompressedLayoutHeirarchy()` which ignores all views which are marked IMPORTANT_FOR_ACCESSIBILITY_NO or IMPORTANT_FOR_ACCESSIBILITY_AUTO (and have been deemed not important by the system), in an attempt to make things less confusing or faster.
+
+**"configurator"** - JSON string (example: `{"method":"setWaitForIdleTimeout","value":5000}`) which sets [UiAutomator Configurator](https://developer.android.com/reference/android/support/test/uiautomator/Configurator.html) timeouts and delays in Android devices. 
```

Comment:
any reason it couldn't just be a JSON object that gets sent rather than a JSON string?